tests/lib/schemas/test_user.py

```diff
@@ -33,6 +34,35 @@ def test_create_user_password_contains_whitespace(db_session):
         CreateUserSchema(username='johndoe', password='pass word')
 
 
+def test_update_user_valid():
+    user = UpdateUserSchema(password='password', password_confirm='password',
+                            old_password='oldpassword')
+
+    assert user.password == 'password'
+    assert user.old_password == 'oldpassword'
+    assert user.password_confirm == 'password'
+
+
+def test_update_user_password_contains_whitespace():
+    with pytest.raises(ValueError,
+                       match="password can't contain whitespaces"):
+        UpdateUserSchema(password='pass word', old_password='testtest',
+                         password_confirm='pass word')
+
+
+def test_update_user_passwords_do_not_match():
+    with pytest.raises(ValueError,
+                       match="passwords must match"):
+        UpdateUserSchema(password='password', old_password='testtest',
+                         password_confirm='wrongpassword')
+
+
+def test_update_user_old_password_missing():
+    with pytest.raises(ValueError,
+                       match="old_password is required when the password field is given"):
```

Comment:
Line length. I'll try to get Codacy to spot these. I'm not a 100% if this PR doesn't contain more of them.